app/models/wakes/location.rb

```diff
@@ -13,7 +14,8 @@ def label
     path
   end
 
-  def url(protocol: 'http', host: ENV['DEFAULT_HOST'])
+  def url(protocol: 'http', host: nil)
+    host ||= self.host || ENV['DEFAULT_HOST']
```

Comment:
@benhutton I ended up using it here: https://github.com/desiringgod/desiring-god-new/blob/develop/lib/scripts/00083_update_facebook_count_on_non_canonical_wakes_locations.rb#L5. It has seemed helpful.